### ISSUE TITLE
RetryableHttpClientTimeout debug message when running CLI

### ISSUE THREAD

**Issue by brian-hamill-bv** [trufflesecurity/trufflehog#1017]
### TruffleHog Version
3.24.0

### Expected Behavior

No debug logs

### Actual Behavior

When running the command:
```
trufflehog github --repo=https://github.com/orgName/repoName --no-verification --no-update --json
```

I am receiving debug output from logrus showing a retryable http client timeout message for the GitHub API:

```
{"fields.level":"debug","level":"info","msg":"[DEBUG] GET https://api.github.com/user","name":"RetryableHttpClientTimeout","time":"2023-01-11T10:04:32Z"}
{"fields.level":"debug","level":"info","msg":"[DEBUG] GET https://api.github.com/user","name":"RetryableHttpClientTimeout","time":"2023-01-11T10:04:33Z"}
```

I have two main questions around this:
Will there be an error message when the maximum retries have been reached/exceeded as defined in [this code?](https://github.com/trufflesecurity/trufflehog/blob/main/pkg/common/http.go#L125)

Is it possible to suppress these debug messages unless running with the `--debug` flag?

### Steps to Reproduce

Run the above command.

## Environment

MacOS Monterey 12.6.1

**Comment by brian-hamill-bv** [comment]
Also seeing this debug output occasionally:
```
{"fields.level":"debug","level":"info","msg":"[DEBUG] GET https://api.github.com/repos/orgName/repoName","name":"RetryableHttpClientTimeout","time":"2023-01-11T12:20:32Z"}
```

**Comment by starsead** [comment]
I have the same problem.

**Comment by dustin-decker** [comment]
My apologies, I merged #995 and it is not correctly respecting the log level. I'll work on a fix.
